### ISSUE TITLE
Game Request: Persona 5 Royal

### ISSUE THREAD

**Issue by GLPG35** [Aethese/Switchence#41]
I did another pull request so, here's the image:

![persona-5-royal](https://user-images.githubusercontent.com/55563130/195356581-ffe9408f-d91d-4cfb-b17f-8735453e78ff.jpg)

**Comment by GLPG35** [comment]
Sorry again, I was just following the rules, I didn't know I could do that.

**Comment by Aethese** [comment]
good point, I'm kind of at fault here. I will change the README to reflect this change I want :)
